src/main/java/appeng/client/render/cablebus/CableBusBakedModel.java

```diff
@@ -122,34 +126,38 @@
 				renderState.getBoundingBoxes(),
 				renderState.getAttachments().keySet(),
 				rand,
-				quads
-		);
+				quads );
 
 		return quads;
 	}
 
 	// Determines whether a cable is connected to exactly two sides that are opposite each other
 	private static boolean isStraightLine( AECableType cableType, EnumMap<EnumFacing, AECableType> sides )
 	{
-		Iterator<EnumFacing> it = sides.keySet().iterator();
+		final Iterator<Entry<EnumFacing, AECableType>> it = sides.entrySet().iterator();
 		if( !it.hasNext() )
 		{
 			return false; // No connections
 		}
-		EnumFacing firstSide = it.next();
-		AECableType firstType = sides.get( firstSide );
+
+		final Entry<EnumFacing, AECableType> nextConnection = it.next();
+		final EnumFacing firstSide = nextConnection.getKey();
+		final AECableType firstType = nextConnection.getValue();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the declaration of "firstType" closer to the code that uses it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1941)